fortinet/package.yaml

```
@@ -0,0 +1,28 @@
+categories:
+- mappings
+- sources
+
+inputs:
+  syslog-url:
+    name: Syslog URL
+    description: The endpoint (including URL scheme) where FortiGate sends syslog.
+    type: string
+    default: udp://0.0.0.0:514
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (2 > 0)

(empty-lines)